### ISSUE TITLE
Datageneration instruct seem don't work

### ISSUE THREAD

**Issue by 66ling66** [frank-xwang/InstanceDiffusion#17]
![屏幕截图 2024-03-18 171430](https://github.com/frank-xwang/InstanceDiffusion/assets/108187964/494834bf-035e-44a9-88cb-6aff6c584cdc)
Could you provide a new instruct? or some example of the training data

**Comment by 66ling66** [comment]
![屏幕截图 2024-03-18 173953](https://github.com/frank-xwang/InstanceDiffusion/assets/108187964/43d9f50a-1b21-4537-8814-1f87afb041f1)

**Comment by frank-xwang** [comment]
Can you provide the full command you used for running the experiments and the error message? Thanks.

**Comment by 66ling66** [comment]
Thanks you,my gpu memory is not encough,and I just change my training strategy.

**Comment by 66ling66** [comment]
[https://github.com/frank-xwang/InstanceDiffusion/blob/dadf0e3b09c2de82bf35b24e3424a14197a29906/trainer.py#L163-L171](url)  I don't want to add this when training the unet model,what should I do?

**Comment by 66ling66** [comment]
https://github.com/frank-xwang/InstanceDiffusion/blob/dadf0e3b09c2de82bf35b24e3424a14197a29906/trainer.py#L163-L171
if I don't want to add this when training the unet model,what should I do? Is there a very simple way to do this?

**Comment by 66ling66** [comment]
https://github.com/frank-xwang/InstanceDiffusion/blob/dadf0e3b09c2de82bf35b24e3424a14197a29906/trainer.py#L167
if I don't want to add this when training the unet model,what shoule I do?Is there a very simple way to do this?

**Comment by frank-xwang** [comment]
Sorry for the late reply. I am a little bit confused, you mean you want to train the model without any grounding inputs? Our model training needs to have instance/part-level location and captioning inputs, otherwise, it should be equivalent to directly fine-tuning the Stable Diffusion model.

**Comment by 66ling66** [comment]
Yes I want to train the model without any grounding inputs,I just train a lora model ,when sampling image,add the grounding inputs,it seems work a litle bit.

**Comment by frank-xwang** [comment]
Oh, I see. Maybe you can provide zero tensors as a placeholder for bbox, masks and instance caption embeddings. The easies way might be calling 'self.grounding_tokenizer_input.get_null_input()'. You should manually set 'self.set=True', and provide the 'self.device', 'self.dtype', 'self.max_box', etc.

**Comment by frank-xwang** [comment]
You can find more details on this function at 'grounding_input/text_grounding_tokinzer_input.py'

**Comment by Chaoriticle** [comment]
> Yes I want to train the model without any grounding inputs,I just train a lora model ,when sampling image,add the grounding inputs,it seems work a litle bit.

Hi, I am a beginner in deep learning and I am very interested in the process you used to train the lora model, can you please share the whole process in detail? It will be very helpful for me, thank you very much!

**Comment by 66ling66** [comment]
> > Yes I want to train the model without any grounding inputs,I just train a lora model ,when sampling image,add the grounding inputs,it seems work a litle bit.
> 
> Hi, I am a beginner in deep learning and I am very interested in the process you used to train the lora model, can you please share the whole process in detail? It will be very helpful for me, thank you very  
First you should do that https://github.com/frank-xwang/InstanceDiffusion/issues/17#issuecomment-2073965484 make sure the grounding inputs is null then you can train the lora model.Maybe you can search what is lora and how to train it.Then all the things solved.
